pkg/kubelet/server/server.go

```diff
@@ -824,15 +824,17 @@ func trimURLPath(path string) string {
 	return parts[0]
 }
 
+var longRunningRequestPathMap = map[string]bool{
```

Comment:
Nit: Can we keep this variable local to `isLongRunningRequest`? We can promote it to package visible later if we have a good reason to.

Reply:
The intention is not to compose the map every time isLongRunningRequest is called.

Reply:
Ah yes great point :)